### PR TITLE
Paper: Spell project names as they spell them

### PR DESCRIPTION
**Pull request by hugovk**
- [ ] I have marked all applicable categories:
    + [ ] exception-raising fix
    + [ ] visual output fix
    + [x] documentation modification
    + [ ] new feature
- [ ] If applicable, I have mentioned the relevant/related issue(s)

Less important but also useful:

- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates
- [ ] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  ```

  [source website]: https://github.com/tqdm/tqdm/
  [known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
  [issue tracker]: https://github.com/tqdm/tqdm/issues?q=

References:
* https://jupyter.org/
* https://matplotlib.org/
* https://pandas.pydata.org/
* https://keras.io/

### CODE REVIEW

**Comment by casperdcl** [comment]
I prefer code versions for everything, in which case we should change *Jupyter* to `jupyter` instead...

File: examples/paper.md [hunk 1]
```diff
@@ -97,11 +97,11 @@ Supported features include:
   terminals
 - Support for custom rendering frontends, including:
     * Command-line interface
-    * *Jupyter* HTML notebooks
-    * `matplotlib`
+    * Jupyter HTML notebooks
```
Comment (lrq3000) [inline examples/paper.md:100]: Should be "Jupyter Notebook" as they spell it, drop "HTML"

**Comment by lrq3000** [comment]
I am agnostic to the use of either the code version or the prose version of the names, as long as the same style is consistently used. Although I have a slight preference for the prose version, since the names are written in the context of a paper, but it doesn't really make a difference in either case IMO.

**Comment by hugovk** [comment]
My preference depends on the context: code style when writing code, or specifically talking about a command; prose style when talking about a project (how the project refers to themselves). 

I agree about prose being more appropriate in the context of a paper.

**Pull request merged**